phpBB/includes/functions_display.php

```diff
@@ -1117,7 +1117,6 @@ function display_custom_bbcodes()
 			'BBCODE_TAG'		=> $row['bbcode_tag'],
 			'BBCODE_TAG_CLEAN'	=> str_replace('=', '-', $row['bbcode_tag']),
 			'BBCODE_HELPLINE'	=> $row['bbcode_helpline'],
```

Comment:
perhaps deprecate and add a BBCODE _TITLE var to be used going forward. WTF does helpline even mean?

Reply:
It's the helpline that is shown when hovering over a BBCode. Commonly: `<description> <usage>`.  example: `Bold text - [b]text[/b]`

And naming it `BBCODE_TITLE` is rather weird again then too, as it is called Helpline everywhere else. And is it not a title, only used in the `title=""` attribute, but that's just for coders and not for the regular John Doe the Admin.

Reply:
How is this not a title? It performs like a tilte, it looks like a title, it smells like a title... 

> It's the helpline that is shown when hovering over a BBCode. Commonly: <description> <usage>. example: Bold text - [b]text[/b]

Thats the definition of a tooltip. I would be ok with that as well over helpline. Semantically to me helpline is just weird and does not make sense. Until you you mentioned whats its used for I had no clue it was even being used there.

Reply:
Wondering how a title smells :P
And a title would be `Bold text`.

It's a line of text helping the user on how to use the BBCode. Which is just located in the `title=""` attribute.

Anyways, I think that might not be for this PR. And to be honest, I think "Helpline" is so integrated with BBCodes at the moment, it might be a bit weird suddenly changing the name.